stl/inc/vector

```diff
@@ -590,7 +739,7 @@ public:
 
 private:
     template <class... _Valty>
-    _CONSTEXPR20 decltype(auto) _Emplace_back_with_unused_capacity(_Valty&&... _Val) {

```

Comment:
Ditto not a fan of drive by changes

Reply:
That was requested by @CaseyCarter since the return type of this function is now unconditionally a `_Ty&`.